docs/spec/spec.md

```diff
@@ -447,6 +447,44 @@ The `rabbitmq:Message` record received can be used to retrieve its contents.
       BasicProperties properties?;
    |};
 ```
+Subtypes of `rabbitmq:AnydataMessage` can be used to bind data to a specific type.
+```ballerina
+    public type StringMessage record {|
+       *rabbitmq:AnydataMessage;
+       string content;
+   |};
+   rabbitmq:Service listenerService =
+   @rabbitmq:ServiceConfig {
+      queueName: "MyQueue"
+   }
+   service object {
+      remote function onRequest(StringMessage message) returns string {
+         return "Hello Back!";
+      }
+   };
+```
+`rabbitmq:BytesMessage` can be used to get the content as a `byte[]` array.
+```ballerina
+# Represents the subtype of `AnydataMessage` record where the message content is a byte array.
+#
+# + content - Message content in bytes
+public type BytesMessage record {|
+    *AnydataMessage;
+    byte[] content;
+|};
+```
+If metadata like `routingKey`, `properties` are not needed, `content` can be directly received as well.
+```ballerina
+   rabbitmq:Service listenerService =
+   @rabbitmq:ServiceConfig {
+      queueName: "MyQueue"
+   }
+   service object {
+      remote function onRequest(string payload) returns string {
+         return "Hello Back!";
```

Comment:
Maybe we can just echo back the payload.

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-rabbitmq/pull/399/commits/3a0c50ad81de8fcbe2dab219b7971de22ca3be2e